src/domains/DevTeam/DevTeam.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+
+import Team from 'components/Team';
+import Loader from 'components/Loader';
+import useEndpoint from 'hooks/useEndpoint';
+import useSnack from 'hooks/useSnack';
+import { Team as TeamType } from 'types';
+import { getDevTeams } from './api';
+
+export default function DevTeam() {
+    const [devTeam, setDevTeam] = React.useState<TeamType[]>([]);
+    const apiRequest = React.useCallback(() => getDevTeams(), []);
+    const [snack] = useSnack();
+    const [sendRequest, isLoading] = useEndpoint(apiRequest, {
+        onSuccess: (results) => {
+            setDevTeam(results.data.devTeam);
+        },
+        onFailure: () =>
+            snack('Something went wrong, please try again', 'error'),
+    });
+    React.useEffect(() => sendRequest(), []);
+
+    return isLoading ? (
+        <div style={{ height: '100vh' }}>
```

Comment:
If using 100 vh, make it such that `overflow: hidden` is also in the style so that we don't get a scroll bar on the loading screen.  Alternatively, you could make the `height: 500px`, which is what I'm doing.  It won't be _perfectly_ centered, but good enough.